### PR TITLE
fix(callout-data): fixed wrong story display on iframe mode

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#3027 

### Description

I've fixed a bug on `CalloutData` story in iframe mode, where spaces on `data` prop, would be turned into unicode strings.

### CODE REVIEW

File: packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js [hunk 1]
```diff
@@ -40,11 +40,24 @@ export default !DDS_CALLOUT_DATA
...
+      let data;
+      if (parameters.props.CalloutData.props.data.includes('%20')) {
+        data = parameters.props.CalloutData.props.data
+          .split('%20')
+          .map(elem => `${elem} `)
+          .join('');
+      } else {
+        data = parameters.props.CalloutData.props.data;
+      }
+      console.log();
```
Comment (jeffchew) [inline packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js:52]: Make sure to remove the `console.log()`

File: packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js [hunk 2]
```diff
@@ -40,11 +40,24 @@ export default !DDS_CALLOUT_DATA
 export const Default = !DDS_CALLOUT_DATA
   ? undefined
   : ({ parameters }) => {
+      let data;
+      if (parameters.props.CalloutData.props.data.includes('%20')) {
+        data = parameters.props.CalloutData.props.data
+          .split('%20')
+          .map(elem => `${elem} `)
+          .join('');
```
Comment (asudoh) [inline packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js:48]: Do we need to care only about space character, or do we need full `decodeURIComponent()`?

Comment (RaphaelAmadeu) [reply packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js:48]: This problem is actually being caused but the percentage mark on the `data`, that if used with a space after it generates a double percentage mark on the url, that causes the bug.

I think that the main problem is the space character, but `decodeURIComponent` seems like a good solution as well

File: packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js (hunk 1 above)
Comment (RaphaelAmadeu) [reply packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js]: Removed that in my latest commit, thanks!

**Pull request merged**